ProcessLib/HydroMechanics/HydroMechanicsProcess-impl.h

```diff
@@ -48,6 +50,27 @@ bool HydroMechanicsProcess<DisplacementDim>::isLinear() const
     return false;
 }
 
+template <int DisplacementDim>
+MathLib::MatrixSpecifications
+HydroMechanicsProcess<DisplacementDim>::getMatrixSpecifications(
+    const int equation_id) const
+{
+    // For the staggered scheme, equation_id == 1 indicates that the matrix
+    // specifications are for the momentum balance equation (deformation).
```

Comment:
this belongs to the `equation_id` parameter doxygen documentation.

Reply:
Replaced this comment with a simple one. Besides, `equation_id` is documented in the header file.